client/components/account-card.jsx

```diff
@@ -81,21 +85,81 @@ export default class AccountCard extends React.Component {
     );
   }
 
+  displayingCommentAndRating() {
+    const currentFiveStarRating = (
+      <div className='flex-center fa-star-holder'>
+        {this.displayingStars(this.state.happyLevel, this.state.currentRating)}
+      </div>
+    );
+    const newFiveStarRating = (
+      <div className='flex-center-column'>
+        {this.displayingNewRating()}
+      </div>
+    );
+
+    const form = (
+      <div className='column-full image-upload-holder flex-center'>
+        <div className='comment-style '>
+          <textarea onChange={this.handleCommentChange} className='comment-input-style'
+                type='textarea'
+                name="comment"
+                autoComplete='off'
+                autoFocus
+                placeholder="Leave a comment" />
+          <div className='column-full'>
+            {newFiveStarRating}
+          </div>
+        </div>
+      </div>
+    );
+
+    if (this.props.hideComment) {
+      return <img src={this.state.photoUrl} />;
+    }
+
+    const img = (
+      <>
+        <div className='column-full image-upload-holder flex-center'>
+          <img src={this.state.photoUrl} />
+        </div>
+        <div className='column-full'>
+          {currentFiveStarRating}
+        </div>
+      </>
+    );
+
+    return this.state.toggleCommentBox ? form : img;
```

Comment:
If possible, we should avoid creating react elements we aren't actually using. Try putting all of these into `if` statements.